app/shop.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+
+class Shop:
+    def __init__(
+            self,
+            name: str,
+            location: list[int, int],
+            products: dict[str: int | float]

```

Comment:
The type hint for the `products` parameter uses a colon `:` instead of a comma `,` to separate the key and value types in the dictionary. It should be `dict[str, int | float]`.